app/assets/javascripts/discourse/app/lib/text-direction.js

```diff
@@ -16,7 +16,14 @@ export function isLTR(text) {
 export function setTextDirections(elem) {
   for (let e of elem.children) {
     if (e.textContent) {
-      e.setAttribute("dir", isRTL(e.textContent) ? "rtl" : "ltr");
+      if (
+        e.tagName.toLowerCase() === "aside" &&
+        e.classList.contains("quote")
```

Comment:
minor ... but tagName is always upper case per spec https://dom.spec.whatwg.org/#ref-for-dom-element-tagname%E2%91%A0 

No need to lower `e.tagName === "ASIDE"` is correct and more efficient anyway.

Reply:
fixed!